docs/gatsby-config.js

```diff
@@ -19,9 +19,9 @@ module.exports = {
           ],
           'Essentials': [
             'essentials/00-queries',
-            'essentials/01-errors',
             'essentials/10-mutations',
             'essentials/20-subscriptions',
+            'essentials/01-errors',
```

Comment:
👍  I was wondering whether this would make more sense. Overall, I initially tried to write the docs as users would write code. Users typically have to handle errors before doing mutations. But this is what the tutorial will be for so 👍 .

PS: because I'm a maniac and like the order of my files in my IDE to be the same as the order in the docs site, I'll rename this file to `21-errors` 🙃